.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+# Notes:
+#  - All software installed below is pinned so they don't get upgraded and break w/o us knowing why.
+#  - If you bump one version up to latest, please bump them all so we stay current!
+
+name: Build medic-couch2pg and run unit test cases
+
+on: [push, pull_request]
+
+jobs:
+  build-and-test:
+    name: Build medic-couch2pg
+    runs-on: ubuntu-18.04
+
+    steps:
+    - name: Checkout branch
+      uses: actions/checkout@v2
+
+    - name: Use nodeJS 10.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+
+    - name: Set environment variables
+      run: |
+        echo "COUCH2PG_SLEEP_MINS=120" >> $GITHUB_ENV
+        echo "COUCH2PG_DOC_LIMIT=1000" >> $GITHUB_ENV
+        echo "COUCH2PG_RETRY_COUNT=5" >> $GITHUB_ENV
+
+    - name: Build
+      run: docker-compose build test
+
+    - name: Run unit-test
```

Minor nitpick... It is preferred that this is a friendly display name:
```suggestion
    - name: Run tests
```

---

Done!